components/create_comment/index.ts

```diff
@@ -98,8 +98,12 @@ function makeMapStateToProps() {
     };
 }
 
-function makeOnUpdateCommentDraft(rootId: string) {
-    return (draft?: PostDraft) => updateCommentDraft(rootId, draft);
+function makeOnUpdateCommentDraft(rootId: string, channelId: string) {
+    return (draft?: PostDraft) => updateCommentDraft(rootId, draft ? {...draft, channelId} : draft);
```

Comment:
@larkox this is used in the callback of either upload success or failure,
 the reasoning is that you might have left that thread, and opened another one since the upload ends.
so the `rootid` should be the one passed in the callback and not the one in the current props.